components/authz-service/server/v2/project.go

```diff
@@ -108,6 +110,10 @@ func (s *state) CreateProject(ctx context.Context,
 	if err != nil {
 		if err == storage_errors.ErrConflict {
 			return nil, status.Errorf(codes.AlreadyExists, "project with ID %q already exists", req.Id)
+		} else if err == storage_errors.ErrMaxProjectsAllowed {
+			return nil, status.Errorf(codes.FailedPrecondition,
+				"max of %d projects allowed while IAM v2 Beta is in progress."
+					"you must delete a project before you can add another one", v2_constants.MaxProjects)
```

Comment:
Does this message fit in the error banner in the UI?

Reply:
i think we should remove that part about deleting a project.. i dont think we want to encourage that because it will orphan resources at least for now, and by the time it wont we will have removed the max.

Reply:
It's much shorter now.